### PR TITLE
BCMOHAD-15804||Mass update changes & performance improvemen msg

### PR DESCRIPTION
**Pull request by ahshaik1**

### CODE REVIEW

File: TPL/force-app/main/default/classes/HCCCostController.cls [hunk 1]
```diff
@@ -258,9 +258,23 @@ public with sharing class HCCCostController {
             hcc.Cost_Include__c = costInclude;
             hcc.Cost_Review__c = costReview;
         }
-        System.enqueueJob(new AccountQueueable(hccList));
+        System.enqueueJob(new HCCQueueable(hccList));
         return currentRecords;
     }
+    @AuraEnabled
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/classes/HCCCostController.cls:264]: Detection process for current running queueable process prevents double-clicking on the process and keeps the user informed that the process is running.

File: TPL/force-app/main/default/classes/HCCQueueable.cls [hunk 2]
```diff
@@ -1,6 +1,6 @@
 public with sharing class HCCQueueable implements Queueable {
     private List<Healthcare_cost__c> hccupdateList;
-    Integer BatchSize = 5000;
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/classes/HCCQueueable.cls:3]: Small queueable batch size proves to be very efficient at 4 seconds per batch. This also prevents salesforce for breaking a large batch into concurrently running batches that can conflict with one another.

File: TPL/force-app/main/default/classes/HCCQueueable.cls [hunk 3]
```diff
@@ -29,6 +29,6 @@ public with sharing class HCCQueueable implements Queueable {
                 System.enqueueJob(followupJob);
             }
         }
-        System.debug(activeBatch);      
+        System.debug(activeBatch);
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/classes/HCCQueueable.cls:32]: Debugs will be removed by the cleanup ticket (already created).

File: TPL/force-app/main/default/lwc/ambulanceRecordsCase/ambulanceRecordsCase.js [hunk 4]
```diff
@@ -295,23 +302,50 @@ export default class AmbulanceRecordsCase extends LightningElement {
     changeCostInclude(event){
         this.costInclude = event.target.checked;
         
+    }
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/lwc/ambulanceRecordsCase/ambulanceRecordsCase.js:305]: The active jobs will be monitored for completion.

**Review by cgijeffolsen — APPROVED** [review]
@ahshaik1 @NataliaNikishina @anudishCGI Approved

**Review by NataliaNikishina — APPROVED** [review]
@ahshaik1 The branch has been successfully validated so i am approving the PR.
cc: @crystalz2022

File: TPL/force-app/main/default/classes/HCCQueueable.cls (hunk 3 above)
Comment (crystalz2022) [reply TPL/force-app/main/default/classes/HCCQueueable.cls:32]: smaller batch size (200 vs 5000) should be helpful

**Review by crystalz2022 — APPROVED** [review]
approved @ahshaik1

**Pull request merged**